Detections/MultipleDataSources/ExchangeServerVulnerabilitiesMarch2021IoCs.yaml

```diff
@@ -190,6 +194,10 @@ entityMappings:
     fieldMappings:
       - identifier: FullName
         columnName: HostCustomEntity
+  - entityType: IP
```

Comment:
How did the new entity got in?

Reply:
@oshezaf it's from a conflict with master